app/v2/models/errors/PersistenceError.scala

```diff
@@ -25,6 +25,7 @@ sealed trait PersistenceError
 object PersistenceError {
   case class MessageNotFound(movementId: DepartureId, messageId: MessageId) extends PersistenceError
   case class DepartureNotFound(departureId: DepartureId)                    extends PersistenceError
+  case class DepartureDoesNotExist(departureId: DepartureId)                extends PersistenceError
```

Comment:
No need to add this, use `DepartureNotFound`

Reply:
That is mapped with 404 in Error Translator

Reply:
so created new one and mapped with 400

Reply:
It's `ErrorTranslator` we need to make a modification to then. We already do something there for an override:

https://github.com/hmrc/common-transit-convention-traders/blob/1ee9f7cb1235f651e4b95fa99d909fb8da3d47b5/app/v2/controllers/ErrorTranslator.scala#L47-L64

and then use it explicitly

https://github.com/hmrc/common-transit-convention-traders/blob/1ee9f7cb1235f651e4b95fa99d909fb8da3d47b5/app/v2/controllers/V2DeparturesController.scala#L135

and so we should do something similar here if we want to override, because it's the controller that cares, not the rest of the stack.

Reply:
As discussed in clarification not required